accord-core/src/main/java/accord/coordinate/Recover.java

```diff
@@ -149,7 +121,8 @@ public boolean hasMetFastPathCriteria()
     private boolean isDone;
 
     final List<RecoverOk> recoverOks = new ArrayList<>();
-    final FastPathTracker<ShardTracker> tracker;
+    final RecoveryTracker tracker;
```

Comment:
🤷  - sometimes it is helpful, but honestly I think I've fallen out of love with `private` except where I really want to enforce it strongly; it's just extra words (a bit like `final` for local variables). I'm king of my package, I can behave nicely. Don't feel strongly about it, happy to make private where relevant.

Reply:
It's pretty low on the list of things I care about, but when I see something a package private I think it's a hint that it is intended to be part of the public (well package public) API of the class and part of how it operate.b

Having things that looks kinda like part of the API, but aren't part of it is the extra noise to me.

Pretty low priority just because most of the time what is meant to be externally used is pretty obvious without the modifiers.

Reply:
Yeah, I understand that and it is sometimes super important for clarity. I’ve changed it here anyway as there was nothing lost with the greater specificity, just musing on why it happened - I think I have come to treat default visibility as acceptable particularly in narrowly-scoped packages, but I’m just reverse engineering (or post hoc rationalising) my actions.